### ISSUE TITLE
questions about your Chrome OS setup

### ISSUE THREAD

**Issue by andreivolt** [srid/mynixos#8]
Hi Srid,

I'm a new Chrome OS user and I'd like to migrate (or rather, adapt) my NixOS config to Crostini's Debian.

I found your repo by searching for NixOS configs for Chrome OS, and I see it mentioned there, which makes me very optimistic, but leaves me with some questions:

- What method of installation did you follow? Did you do a user-mode installation, or (don't think it's possible, but I wish) some sort of system installation (like nixos-in-place?). Ideally, could you kindly `grep` your shell history for the relevant commands so that I get an idea of your setup?
- Is there anything about it you don't like (compared to an actual NixOS install)?

Thanks!

**Comment by srid** [comment]
Hi!

I don't have NixOS installed on my crostini container. I continue to use Debian but use Nix (not NixOS) to install packages. `home.nix` is the only file I share with this crostini setup. It requires `home-manager` to be installed on the crostini container. Have you tried following the instructions here? https://github.com/srid/mynixos#on-chromeos

**Comment by andreivolt** [comment]
Thanks for your reply! I guess I somehow overestimated the complexity of the setup, yes I had read that part and I actually used to use `home-manager` on NixOS before I abandoned it due to some partly self-created complexity creep (countless custom modules for everything). I might have been wrong in retrospect, but I digress.

So, yeah, I just have to do a user-mode install and install `home-manager` and I'm done. Awesome!

Thanks again and have a nice day
